apps/modeler/src/cloud_mesh.cpp

```diff
@@ -117,8 +117,8 @@ pcl::modeler::CloudMesh::save(const std::vector<const CloudMesh*>& cloud_meshes,
     if (filename.rfind(".obj") == (filename.length() - 4)) {
       std::size_t delta = cloud_mesh.cloud_->size();
```

Comment:
Update to use `index_t`? I don't know the type of `cloud_` but that'd remove the cast from the loop